### PR TITLE
Support MCG S3 endpoint

### PR DESCRIPTION
**Pull request by sgilra**

### CODE REVIEW

**Review by jtruskow — CHANGES_REQUESTED** [review]
Other than the 1 requested change -- export on different MinIO cluster and import to S3 cluster was successful. Customizations worked.

Only tested STT Customization (but TTS should work as well)

File: import_export.sh [hunk 1]
```diff
@@ -334,15 +374,15 @@ elif [ ${COMMAND} = 'import' ] ; then
...
     echo "----- Importing MinIO Database -----"
-    start_minio_port_forward $MINIO_SVC $MINIO_LPORT $MINIO_PORT $TMP_FILENAME
+    start_os_port_forward $OS_SVC $OS_LPORT $OS_PORT $OS_NS $TMP_FILENAME
 
-    $MC --insecure config host add speech-minio https://localhost:$MINIO_LPORT ${MINIO_ACCESS_KEY} ${MINIO_SECRET_KEY}
+    $MC --insecure config host add speech-minio https://localhost:$OS_LPORT ${OS_ACCESS_KEY} ${OS_SECRET_KEY}
     cmd_check
 
-    $MC --insecure cp -r ${EXPORT_DIR}/minio/stt-customization-icp/customizations speech-minio/stt-customization-icp
+    $MC --insecure cp -r ${EXPORT_DIR}/minio/${STT_CUST_BUCKET}/customizations speech-minio/${STT_CUST_BUCKET}
```
[the remark this answers is not in the document]
Comment (sgilra) [reply import_export.sh:382]: LGTM

**Comment by sgilra** [comment]
> Other than the 1 requested change -- export on different MinIO cluster and import to S3 cluster was successful. Customizations worked.
> 
> Only tested STT Customization (but TTS should work as well)

Shouldn't the tts-customization and stt-async be empty and skipped? There is no reference to these busckets being exported or imported in the script.

**Comment by jtruskow** [comment]
> > Other than the 1 requested change -- export on different MinIO cluster and import to S3 cluster was successful. Customizations worked.
> > Only tested STT Customization (but TTS should work as well)
> 
> Shouldn't the tts-customization and stt-async be empty and skipped? There is no reference to these busckets being exported or imported in the script.

I did see as part of the import it attempted to import TTS-cust and STT-async despite these services not being installed and marked failed as a result (though stt-cust was successfully imported and worked)

This is something we should fix but imo it makes sense to do it when we move to the common CP4D framework

**Review by jtruskow — APPROVED** [review]

**Comment by sgilra** [comment]
I think that is done for tts-cust and asyc postgres export/import not minio/s3

**Comment by sgilra** [comment]
Also please merge, I don't have the permission to do so.

**Pull request merged**